addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -220,16 +220,41 @@ odoo.define('website.s_website_form', function (require) {
 
             // Prepare form inputs
             this.form_fields = this.$target.serializeArray();
+            var too_many_files = false
+            var file_too_big = false
```

Comment:
- Use `const` or `let` instead of `var` (most of the time `const`)
- Once you will have installed a linter, you will never forget about the end-of-instructions `;` anywore :)